staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -410,6 +412,7 @@ func init() {
 		describeServiceAccount,
 		describeStatefulSet,
 		describeStorageClass,
+		describeVolumeAttributesClass,
```

Comment:
Where is the func describeVolumeAttributesClass similar to https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/describe/describe.go#L4655 ?

Reply:
Hi @sunnylovestiramisu, I'm not sure I understand your question. Can you please clarify it? Thanks!

IMO, the `describeVolumeAttributesClass` function is similar to the `describeVolumeAttributes` function in the link you provided. They both print the parameters and driver information.

Reply:
Where did you add the function definition for describeVolumeAttributesClass?

Reply:
@sunnylovestiramisu https://github.com/kubernetes/kubernetes/pull/122640/files#diff-a4c0ef375b50e760791f9207a7660b39093adc8617a12e523e48c0f5ba160e6dR4710

Reply:
Oh my bad somehow search that string on this page does not show that.